src/test/resources/com/stripe/model/charge_outcome.json

```diff
@@ -1,5 +1,6 @@
 {
     "network_status": "approved_by_network",
+    "risk_level": "normal",
```

Comment:
@brandur can't believe I read it 3 times to ensure it was the right order! Fixed!